doc/3.Algorithms.md

```diff
@@ -184,7 +184,7 @@ Assertion 3.4.*.
 
 Assertion 3.4.*.
 
-13. Requester -> NEGOTIATE_ALGORITHMS {SPDMVersion=NegotiatedVersion, ExtAlgCount_0=0xF, ExtAlgCount_1=0xF, ...} -- if NegotiatedVersion=1.1+
+13. Requester -> NEGOTIATE_ALGORITHMS {SPDMVersion=NegotiatedVersion, ExtAlgCount_0=0x2F, ExtAlgCount_1=0x2F, ...} -- if NegotiatedVersion=1.1+

```

Comment:
It should be `FixedAlgCount=2, ExtAlgCount=0xF`

Reply:
Thanks. I have corrected it.